package.json

```diff
@@ -37,6 +37,7 @@
     "@changesets/changelog-github": "^0.4.0",
     "@changesets/cli": "^2.17.0",
     "@manypkg/cli": "^0.19.1",
+    "@parcel/packager-ts": "2.7.0",
```

Comment:
I don't see this referenced anywhere else. Is this meant to be here?

Reply:
It's part of parcels auto install thing. I'll double check though.